reports/Washington DC.md

```diff
@@ -811,3 +811,23 @@ geolocation: 38.9624407, -77.0286733
 
 * https://twitter.com/FordFischer/status/1321905855281958913
 
+
+### Police make violent arrests | November 3rd
+
+Violent arrests made by MPD during an Election day protest at Black Lives Matter plaza.
+
+tags: arrest, grab, protester, push, shove, tackle
+
+id: dc-dc-50
+
+geolocation: 38.9013441, -77.0387466
+
+**Links**
+
+* https://twitter.com/itakeshots1/status/1323760660044488706
+* https://twitter.com/itakeshots1/status/1323760754105978886
+* https://twitter.com/itakeshots1/status/1323828366806523904
+* https://twitter.com/duckdiver19/status/1323759348808474624
```

Comment:
This link is dead; account was suspended

Reply:
removed, thanks